02/app.js

```diff
@@ -4,6 +4,6 @@ document.addEventListener("DOMContentLoaded", () => {
     const handleMouseMove = e => console.log("mousemove");
     const handleMouseLeave = e => console.log("mouseleave");
 
-    divElement.addEventListener("mousemove", handleMouseMove);
-    divElement.addEventListener("mouseleave", handleMouseLeave);
+    divElement && divElement.addEventListener("mousemove", handleMouseMove);
+    divElement && divElement.addEventListener("mouseleave", handleMouseLeave);
```

Comment:
W tym przypadku już bym zastosował normalnego if-a czytelniej i bezpieczniej bo jeśli coś zrobię z divElement to nie będę musiał tego zmieniać w obu miejscach :P